src/components/ReloadInfo.tsx

```diff
@@ -9,6 +9,8 @@ const ReloadInfo: FC = () => {
     localStorage.setItem('showReloadInfo', '0')
   }, [])
 
+  setTimeout(handleDismiss, 10000)
```

Comment:
You need to put that into a `useEffect` hook to prevent setting the timeout again when the component will be re-rendered.

```suggestion
  useEffect(() => {
    const timeoutId = setTimeout(() => {
      handleDismiss()
    }, 10000)

    return () => clearTimeout(timeoutId)
  }, [])
```

Reply:
Ah thanks, makes sense! Changed now.

I also changed it to only call `setShowReloadInfo(false)` and not `handleDismiss` because the latter persists `showReloadInfo = 0` in the local storage.

Reply:
Ah thanks, makes sense! Changed now.

I also changed it to only call `setShowReloadInfo(false)` and not `handleDismiss` because the latter persists `showReloadInfo = 0` in the local storage.